### PR TITLE
[SEED-909] Use seed-services-client for utils.py

### PR DESCRIPTION
**Pull request by alexmuller**
There are 2 uses here which involve pagination which I think will be changed as part of @KaitCrawford's pagination work so I've marked them with `FIXME`s for now.

### CODE REVIEW

File: hellomama_registration/utils.py [hunk 1]
```diff
@@ -89,24 +105,17 @@ def get_identity(identity):
...
-    }
-    r = session.get(url, params=params, headers=headers).json()
-    if len(r["results"]) > 0:
-        return r["results"][0]["address"]
-    else:
-        return None
+    return identity_store_client.get_identity_address(identity)
 
 
-def search_identities(params):
```
Comment (KaitCrawford) [inline hellomama_registration/utils.py:106]: Why the change to the arguments here?

Comment (alexmuller) [reply hellomama_registration/utils.py:106]: Ah sorry I put it in the commit message but didn't copy it into the pull request:

This behaviour more closely matches the seed-services API client so it will make it easier to use the API client when that supports pagination.

**Review by KaitCrawford — APPROVED** [review]
:+1:
